#! /bin/sh /usr/share/dpatch/dpatch-run
## libpri_bristuffed.dpatch by Tzafrir Cohen <tzafrir.cohen@xorcom.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: use libpri-bristuffed instead of libpri for building chan_zap.
## DP: libpri-bristuffed is a debian-specific name for a copy of libpri 
## DP: with the bristuff patch.

@DPATCH@
Index: asterisk-1.2.24.dfsg+pf.0.1~svn1329/channels/Makefile
===================================================================
--- asterisk-1.2.24.dfsg+pf.0.1~svn1329.orig/channels/Makefile	2007-08-16 12:56:45.000000000 +0200
+++ asterisk-1.2.24.dfsg+pf.0.1~svn1329/channels/Makefile	2007-08-16 12:56:49.000000000 +0200
@@ -102,10 +102,15 @@
   CHANNEL_LIBS+=chan_alsa.so
 endif
 
+ifneq ($(wildcard $(CROSS_COMPILE_TARGET)/usr/lib/libpri-bristuffed.so.1),)
+  CFLAGS+=-DZAPATA_PRI -I/usr/include/bristuffed
+  ZAPPRI=-lpri-bristuffed
+else
 ifneq ($(wildcard $(CROSS_COMPILE_TARGET)/usr/lib/libpri.so.1)$(wildcard $(CROSS_COMPILE_TARGET)/usr/local/lib/libpri.so.1),)
   CFLAGS+=-DZAPATA_PRI
   ZAPPRI=-lpri
 endif
+endif
 
 ifneq ($(wildcard $(CROSS_COMPILE_TARGET)/usr/lib/libgsmat.so.1)$(wildcard $(CROSS_COMPILE_TARGET)/usr/local/lib/libgsmat.so.1),)
   CFLAGS+=-DZAPATA_GSM
